modules/efk-logging-curator-delete-index.adoc

```diff
@@ -3,13 +3,13 @@
 // * logging/efk-logging-curator.adoc
 
 [id="efk-logging-curator-yaml-{context}"]
-= Deleting Elasticsearch data
+= Configuring Curator index deletion
 
 ////
 This file can be expanded when more functions are supported. Only delete_indices currently. When more functions are added, we can use include::modules/efk-logging-curator-actions.adoc[leveloffset=+1] for a strict delete index files topic, if needed
 ////
 
-You can configure Curator to delete Elasticsearch data based on retention settings.
+You can configure Curator to delete Elasticsearch data based on retention settings. You can specify specific projects to delete indices and set default values for any projects not specified.
```

Comment:
instead of `You can ... specified.` say `You can configure per-project and global settings.  Global settings apply to any project not specified.  Per-project settings override global settings.`